### PR TITLE
[AN-4565] Adds basic profile attributes

### PR DESCRIPTION
**Pull request by shumengye**
#### Description

- Removes setting of tracking ID from backend
- Adds anonymous profile attributes for registration week and flag if user has contact

#### Ticket
 https://wearezeta.atlassian.net/browse/AN-4565
#### APK
[Download build #7727](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7727/artifact/build/artifact/wire-dev-PR241-7727.apk)
[Download build #7735](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7735/artifact/build/artifact/wire-dev-PR241-7735.apk)
[Download build #7736](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7736/artifact/build/artifact/wire-dev-PR241-7736.apk)
[Download build #7737](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7737/artifact/build/artifact/wire-dev-PR241-7737.apk)
[Download build #7743](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7743/artifact/build/artifact/wire-dev-PR241-7743.apk)
[Download build #7744](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7744/artifact/build/artifact/wire-dev-PR241-7744.apk)
[Download build #7746](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7746/artifact/build/artifact/wire-dev-PR241-7746.apk)

### CODE REVIEW

**Comment by shumengye** [comment]
build again

File: wire-core/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -795,6 +795,7 @@
     <string name="timestamp_pattern__separator_header__yesterday">Yesterday</string>
     <string name="timestamp_pattern__24h_format">HH:mm</string> <!-- 23:15 -->
     <string name="timestamp_pattern__12h_format">hh:mm a</string>
+    <string name="timestamp_pattern__week">w, YYYY</string>
```
Comment (hakonbo) [inline wire-core/src/main/res/values/strings.xml]: should add translatable=false

**Comment by hakonbo** [comment]
build again

**Comment by hakonbo** [comment]
build again

**Pull request merged**